index.html

```diff
@@ -139,21 +139,24 @@ <h2>Skills</h2>
                 <i class="fa-sharp fa-solid fa-angle-right"></i>
               </div>
               <hr>
-            </li>
-            
+            </li> 
         </ul>
     </section>
-    <section class="contact-me">
-        <div class="flex-contact">
-            <h2>Contact me</h2>
-        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-        <form action="" id="contact-form">
-            <input type="text" value="Mudasir">
-            <input type="email" name="" id="" value="muda@nonihub.org">
-            <textarea name="" id="" cols="30" rows="10" form="contact-form" placeholder="Write your message here"></textarea>
-            <input class="submit-btn" type="submit" value="Get In Touch">
-        </form>
-        </div>
+    <section class="contact-area">
+        <div class="contact-me">
+            <h1>Contact Me</h1>
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly make sure you use a `<hx>` tag other than `<h1>`  (e.g.: `<h2>`, `<h3>`,...) for semantics and accessibility reasons. Your project is supposed to only have one `<h1>`.